msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/ui/PrivateMessageManagerImpl.java

```diff
@@ -2154,12 +2157,24 @@ public PrivateMessage getPvtMsgReplyMessage(PrivateMessage currentMessage, MimeM
 
 	  PrivateMessage rrepMsg = createResponseMessage(currentMessage, msg, from);
 
-	  if (StringUtils.isNotBlank(bodyBuf[0].toString())) {
-		  bodyBuf[0].insert(0, "<pre>");
-		  bodyBuf[0].insert(bodyBuf[0].length(), "</pre>");
-		  rrepMsg.setBody(bodyBuf[0].toString());
+	  
+      StringBuilder alertMsg = new StringBuilder();
+      StringBuilder cleanBody;
+	  if (StringUtils.isNotBlank(bodyBuf[1].toString())) {
+		  cleanBody = new StringBuilder(formattedText.processFormattedText(bodyBuf[1].toString(), alertMsg));
 	  } else {
-		  rrepMsg.setBody(bodyBuf[1].toString());
+		  cleanBody = new StringBuilder(formattedText.escapeHtml(bodyBuf[0].toString()));
+		  if(StringUtils.isNotBlank(cleanBody)) {
+			  cleanBody.insert(0, "<pre>");
+			  cleanBody.insert(cleanBody.length(), "</pre>");
+		  }
+	  }
+	  
+	  if(StringUtils.isBlank(cleanBody)) {
+		  log.warn("Exception: Unexpected error processing text of body.");
```

Comment:
improve the message here and remove the word Exception

Reply:
Resolved